pom.xml

```diff
@@ -64,7 +64,7 @@
     <java.version>11</java.version>
 
     <chicory.version>1.1.0</chicory.version>
-    <junit.version>3.8.1</junit.version>
+    <junit.version>4.13.2</junit.version>
```

Comment:
let's keep this in a separate PR (actually if you want to migrate all to junit5 let me know, that'd be welcome :D)

Reply:
No problem :)